android/javatests/org/chromium/chrome/browser/BytecodeTest.java

```diff
@@ -2641,6 +2641,7 @@ private boolean methodExists(
             return false;
         }
         for (Method m : c.getDeclaredMethods()) {
+            boolean tryAnotherMethod = false;
```

Comment:
maybe `didMethodParamsMatch` would be a little more clear here?

Reply:
I agree, the proposed name is more clear when reading the code, thanks

modified at a fixup https://github.com/brave/brave-core/pull/29767/commits/caacdd1938c186e4680a6738e060ca058469b765